### PR TITLE
Property "children" does not have setter in class CategoryInterface #10088

### PR DESCRIPTION
**Pull request by devamitbera**
#10088

### CODE REVIEW

File: app/code/Magento/Catalog/Api/Data/CategoryInterface.php [hunk 1]
```diff
@@ -143,6 +143,15 @@ public function setLevel($level);
     public function getChildren();
 
     /**
+     * Set category children
+     *
+     * @param string $children
+     * @return $this
+     */
+    public function setChildren($children);
```
Comment (sidolov) [inline app/code/Magento/Catalog/Api/Data/CategoryInterface.php:151]: Unfortunately, we are not able to add new methods to the existing interfaces marked as api according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

**Review by sidolov — DISMISSED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5163](https://jira.corp.magento.com/browse/ENGCOM-5163) has been created to process this Pull Request

**Review by sidolov — CHANGES_REQUESTED** [review]
PR contains breaking changes

**Comment by devamitbera** [comment]
@magento-engcom-team,do you need anything else from my end,then please let me know.

**Comment by sidolov** [comment]
@devamitbera , please, take a look at review comment.

**Comment by sidolov** [comment]
@devamitbera , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Comment by devamitbera** [comment]
@sidolov 

Sorry, I was very.

I wish to continue to work with this issue.

Assigning myself again.

**Pull request closed without merging**